### PR TITLE
added support to pass argument to  differential dataflow

### PR DESCRIPTION
**Pull request by PramodBisht**
related to #25 
For a now, this is implemented only for worker argument of timely.
@nikomatsakis  please check.

### CODE REVIEW

File: src/output/naive.rs [hunk 1]
```diff
@@ -22,16 +22,21 @@ use std::sync::Mutex;
...
 
     // Use a channel to send `all_facts` to one worker (and only one)
     let (tx, rx) = mpsc::channel();
     tx.send(all_facts).unwrap();
     mem::drop(tx);
     let rx = Mutex::new(rx);
+    let mut dataflow_arg = Vec::new();
+    if workers > 1 {
+        let item = format!("{}{}", "w", workers);
+        dataflow_arg.push(item);
```
Comment (nikomatsakis) [inline src/output/naive.rs]: I think this should be

```
dataflow_arg.push(format!("-w"));
dataflow_arg.push(format!("{}", workers));
```

though one thing may work too

File: src/output/timely_opt.rs [hunk 2]
```diff
@@ -47,8 +47,13 @@ pub(super) fn compute(dump_enabled: bool, mut all_facts: AllFacts) -> Output {
     tx.send(all_facts).unwrap();
     mem::drop(tx);
     let rx = Mutex::new(rx);
+    let mut dataflow_arg = Vec::new();
+    if workers > 1 {
+        let item = format!("{}{}", "w", workers);
+        dataflow_arg.push(item);
```
Comment (nikomatsakis) [inline src/output/timely_opt.rs]: same here; maybe we want a helper fn :)

Comment (PramodBisht) [reply src/output/timely_opt.rs]: for helper function, any ideal place where I should store this? or should I create common helper functions file for that?

File: src/output/naive.rs (hunk 1 above)
Comment (PramodBisht) [reply src/output/naive.rs]: done!

File: src/output/timely_opt.rs (hunk 2 above)
Comment (PramodBisht) [reply src/output/timely_opt.rs]: @nikomatsakis done!

**Pull request merged**